src/inc/corinfo.h

```diff
@@ -977,8 +977,9 @@ enum CorInfoIntrinsics
 enum InfoAccessType
 {
     IAT_VALUE,      // The info value is directly available
-    IAT_PVALUE,     // The value needs to be accessed via an       indirection
-    IAT_PPVALUE     // The value needs to be accessed via a double indirection
+    IAT_PVALUE,     // The value needs to be accessed via an         indirection
+    IAT_RELPVALUE,  // The value needs to be accessed via a relative indirection
```

Comment:
This is changing JIT/EE interface definition. The JIT/EE interface definition will need new GUID.

Reply:
It would be better to add `IAT_RELPVALUE` at the end of the enum rather than in the middle.
It less of a breaking change that way.